package.json

```diff
@@ -1,139 +1,146 @@
+        "aliases": [
+          "V"
         ],
-        "grammars": [
-            {
-                "language": "v",
-                "scopeName": "source.v",
-                "path": "./syntaxes/v.tmLanguage.json"
-            }
+        "extensions": [
+          ".v",
+          ".vh",
+          ".vsh"
         ],
-        "configuration": {
-            "title": "V",
-            "properties": {
-                "v.format.enable": {
-                    "scope": "resource",
-                    "type": "boolean",
-                    "default": false,
-                    "description": "Enable/disable the V formatter (not yet implemented)."
-                }
-            }
-        },
-        "configurationDefaults": {
-            "[v]": {
-                "editor.insertSpaces": false,
-                "editor.formatOnSave": true,
-                "editor.codeActionsOnSave": {
-                    "source.organizeImports": true
-                }
-            }
-        },
-        "commands": [
-            {
-                "command": "v.run",
-                "title": "V: Run current file"
-            },
-            {
-                "command": "v.prod",
-                "title": "V: Build an optimized executable from current file"
-            },
-            {
-                "command": "v.help",
-                "title": "V: Show help info"
-            },
-            {
-                "command": "v.ver",
-                "title": "V: Show version info"
-            },
-            {
-                "command": "v.path",
-                "title": "V: Show local paths info"
-            },
-            {
-                "command": "v.test.file",
-                "title": "V: Test current file"
-            },
-            {
-                "command": "v.test.package",
-                "title": "V: Test current package"
-            },
-            {
-                "command": "v.playground",
-                "title": "V: Upload and share current code to V playground"
-            }
-        ]
-    },
-    "activationEvents": [
-        "onLanguage:v",
-        "onCommand:v.run",
-        "onCommand:v.prod",
-        "onCommand:v.help",
-        "onCommand:v.ver",
-        "onCommand:v.path",
-        "onCommand:v.test.file",
-        "onCommand:v.test.package",
-        "onCommand:v.playground"
+        "configuration": "./language-configuration.json"
+      }
     ],
-    "main": "./out/extension.js",
-    "dependencies": {
-        "vscode-languageclient": "~5.2.1"
+    "grammars": [
+      {
+        "language": "v",
+        "scopeName": "source.v",
+        "path": "./syntaxes/v.tmLanguage.json"
+      }
+    ],
+    "configuration": {
+      "title": "V",
+      "properties": {
+        "v.format.enable": {
+          "scope": "resource",
+          "type": "boolean",
+          "default": false,
+          "description": "Enable/disable the V formatter (vfmt)"
+        },
+        "v.format.args": {
```

Comment:
Most of these changes are just from prettier running. This `v.format.args` thing is new though